requirements.in

```diff
@@ -22,4 +22,5 @@ flask_wtf==0.14.3
 pgpy==0.5.4
 cbor==1.0.0
 mnemonic==0.20
-cryptography==3.4.7
\ No newline at end of file
+mkdocs==1.2.3
```

Comment:
Do we need to have it in main requirements? Can't we move it to dev requirements instead?

Reply:
Removed it. It's now on the fly loaded in the utils-wrapper script-